src/sentry/templates/sentry/debug/mail/preview.html

```diff
@@ -63,6 +63,9 @@
       <option value="html">HTML</option>
       <option value="txt" {% if format == 'txt' %} selected="selected"{% endif %}>Plaintext</option>
     </select>
+    <form action="" method="post" style="margin-left: auto; margin-block-end: 0;">
+      <button type="submit">Send</button>
```

Comment:
@Neo-Zhixing what about all the other email previews that don't extend `MailPreviewView`?

Reply:
> This will be addressed in a future PR.